internal/bminventory/inventory.go

```diff
@@ -5066,6 +5061,8 @@ func (b *bareMetalInventory) V2UpdateHostIgnitionInternal(ctx context.Context, p
 			log.WithError(err).Errorf("Failed to parse host ignition config patch %s", params.HostIgnitionParams)
 			return nil, common.NewApiError(http.StatusBadRequest, err)
 		}
+	} else {
+		log.Infof("Applying empty host ignition config to host %s in infra-env %s", params.HostID, params.InfraEnvID)
```

Comment:
please add a comment that it means that it will clear the previous configuration, maybe it's just me but it's not clear from this log message.

Reply:
Rewritten the log line to make it clear